src/components/top-bar/styles.ts

```diff
@@ -3,6 +3,17 @@ import { isDesktop, isDesktopOrBelow, isTablet, isTabletOrBelow } from '../../st
 import { ElButtonGroup } from '../button-group'
 import { css } from '@linaria/core'
 
+export const elReapitLogo = css`
+  // need to copy width and height here otherwise it will automatically set to 1em
```

Comment:
yep that is works, thanks 👍  

> todo: Can you avoid using ?react when importing the Reapit SVG and remove this class? 🙏 I've left a comment later in this PR describing how to avoid ?react...

Sure